src/main/java/helloworld/constant/Constants.java

```diff
@@ -0,0 +1,9 @@
+package helloworld.constant;
+
+public final class Constants {
```

Comment:
A class, in order to be a truly immutable class, should also define a private constructor, preventing its instantiation. Was it the idea?